health-events-analyzer/pkg/reconciler/recovery.go

```
@@ -0,0 +1,1316 @@
+}
+
+func (r *Reconciler) currentDerivedStatesForNode(
+	ctx context.Context,
+	rule config.HealthEventsAnalyzerRule,
+	nodeName string,
+) ([]recoveryTarget, error) {
+	cursor, err := r.databaseClient.Find(ctx, r.recoveryLookupFilter(agentName, rule.Name, nodeName), nil)
+	if err != nil {
+		return nil, err
+	}
+	defer cursor.Close(ctx)
```

golangci-lint (2.13.2)

[error] Error return value of `cursor.Close` is not checked

(errcheck)